portfolio/src/main/webapp/script.js

```diff
@@ -29,8 +29,17 @@ hideables.forEach(hideable => {
   h3.addEventListener('click', () => ul.classList.toggle("hidden"));
 });
 
-async function getGreeting() {
-    const response = await fetch("/data");
-    const quote = await response.text();
-    document.getElementById("quote-container").innerHTML = quote;
+async function getComments() {
+  
+  const response = await fetch("/data");
+  const text = await response.text();
+  const comments = JSON.parse(text);
+  
+  const formattedComments = comments.map((comment) => {
+    return `<p>${comment.name} said: ${comment.text}</p>`
```

Comment:
if you omit the curly brackets, you can remove the `return`. Additionally, you can omit the parameter parentheses in a lambda function if only one parameter is given. 
```javascript
comments.map(comment => `<p>${comment.name} said: ${comment.text}</p>`)
```